src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DocumentVersionCache.cs

```diff
@@ -7,8 +7,8 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer
 {
     internal abstract class DocumentVersionCache : ProjectSnapshotChangeTrigger
     {
-        public abstract bool TryGetDocumentVersion(DocumentSnapshot documentSnapshot, out long version);
+        public abstract bool TryGetDocumentVersion(DocumentSnapshot documentSnapshot, out int? version);
 
-        public abstract void TrackDocumentVersion(DocumentSnapshot documentSnapshot, long version);
+        public abstract void TrackDocumentVersion(DocumentSnapshot documentSnapshot, int? version);
```

Comment:
Do we ever pass in a `null` version to `TrackDocumentVersion`? Doesn't feel right for this to be nullable is why I ask